spec/index.bs

```diff
@@ -1805,21 +1833,25 @@ If any of the following conditions are violated:
         |view|'s texture's {{GPUTextureDescriptor/sampleCount}} must be 1.
     1. If |layoutBinding|.{{GPUBindGroupLayoutBinding/type}} is
         {{GPUBindingType/"sampled-texture"}}, |view|'s texture's {{GPUTextureDescriptor/usage}}
-        must include {{GPUTextureUsage/SAMPLED}}.
+        must include {{GPUTextureUsage/SAMPLED}}. The texture's [=subresources=] seen by the |view|
+        are considered to be [=used=] by the resulting bind group as {{GPUTextureUsage/SAMPLED}}.
```

Comment:
We could make this more formal (put the subresource usages in a list, or something), but it can definitely be done later